apps/launch/common-util/constants/stakingContract.ts

```diff
@@ -51,6 +52,7 @@ export const IMPLEMENTATION_ADDRESSES: Addresses = {
   [base.id]: '0xEB5638eefE289691EcE01943f768EDBF96258a80',
   [arbitrum.id]: '0x04b0007b2aFb398015B76e5f22993a1fddF83644',
   [celo.id]: '0xe1E1B286EbE95b39F785d8069f2248ae9C41b7a9',
+  [mode.id]: '0xE49CB081e8d96920C38aA7AB90cb0294ab4Bc8EA',
```

Comment:
@kupermind would be great of you could check all added addresses on this page

Reply:
Just FYI this is confusing for us :). We call this `StakingTokenAddress` :).